packages/cluster-settings/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@k8slens/cluster-settings",
+  "version": "6.5.0-alpha.1",
+  "description": "Injection token exporter for cluster settings configuration",
+  "license": "MIT",
+  "private": false,
+  "mode": "production",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "clean": "rimraf dist/",
+    "generate-types": "tsc --d --declarationDir ./dist --declarationMap --emitDeclarationOnly",
+    "build": "npm run generate-types && swc ./src/index.ts -d ./dist"
+  },
+  "devDependencies": {
```

Comment:
Should we include `typescript` as dev dependency as well?

Reply:
Not sure. It works well without specify typescript here. Probably it uses some globally installed typescript somehow.

Reply:
Yes, try to remove global ‘typescript’ package and check again.